chart/kubeapps/templates/_helpers.tpl

```diff
@@ -230,11 +230,11 @@ kubeapps: ingress.tls
       {{- range $plugin, $options := .Values.packaging }}
         {{- if $options.enabled }}
           {{- if eq $plugin "carvel" }}
-            {{- $enabledPlugins = append $enabledPlugins "kapp-controller-packages" }}
+            {{- $enabledPlugins = append $enabledPlugins "kapp-controller" }}
```

Comment:
These changes will be causing a failure in CI. We updated our local repo to use the `-packages` suffix for those plugins as it is better in the long term. We plan to update the bitnami chart on the next tagged release.

Reply:
I've pushed a commit reverting this.